account/identity.go

```diff
@@ -292,7 +292,7 @@ func IdentityFilterByUserID(userID uuid.UUID) func(db *gorm.DB) *gorm.DB {
 // IdentityFilterByUsername is a gorm filter by 'username'
 func IdentityFilterByUsername(username string) func(db *gorm.DB) *gorm.DB {
 	return func(db *gorm.DB) *gorm.DB {
-		return db.Where("username = ?", username).Limit(1)
+		return db.Where("lower(username) = lower(?)", username).Limit(1)
```

Comment:
@xcoulon, please change to 

```go
return db.Where("lower(username) = ?", strings.ToLower(username)).Limit(1)
```

just like here: https://github.com/fabric8-services/fabric8-wit/pull/1491/files#diff-63bbabb366f733abdd1483ac55ce4436R367